consensus/ConsensusTableUpdater.py

```diff
@@ -0,0 +1,338 @@
+import progressbar, sys
+from molgenis import client as molgenis
+from yaspin import yaspin
+from termcolor import colored, cprint
+from consensus.DataRetriever import DataRetriever
+from consensus.ConsensusTableGenerator import ConsensusTableGenerator
+from consensus.MolgenisConfigParser import MolgenisConfigParser as ConfigParser
+from consensus.HistorySorter import HistorySorter
+from consensus.ConsensusReporter import ConsensusReporter
+
+
+class ConsensusTableUpdater:
+    """The ConsensusTableUpdater uploads consensus and consensus comments by first creating a csv file for them"""
+
+    def __init__(self, data, tables, molgenis_server):
+        """
+        :param data: a dictionary with:
+            - consensus_data: variant information as created by process_variants in ConsensusTableGenerator
+            - lab_classifications: all lab classifications as all_lab_classifications in ConsensusTableGenerator
+            - history: the history data
+        :param tables: a dictionary with:
+            - consensus_table: the fully qualified name of the consensus table
+            - comments_table: the fully qualified name of the consensus comments table
+        :param previous_exports: the id of the previous exports (format yymm, for instance: 1810)
+        :param molgenis_server: logged in molgenis client
+        """
+        self.molgenis_server = molgenis_server
+        consensus_table = tables['consensus_table']
+        comments_table = tables['comments_table']
+        consensus_data = data['consensus_data']
+        lab_classifications = data['lab_classifications']
+        self.history = data['history']
+        self.consensus_table = consensus_table
+        self.comments_table = comments_table
+        self.delete_consensus()
+        self.delete_comments()
+        self.upload_comments(consensus_data)
+        file_name = self.produce_consensus_csv(consensus_data, lab_classifications)
+        self.upload_consensus(file_name, len(consensus_data))
+
+    @staticmethod
+    def create_consensus_header(labs):
```

Comment:
labs should probably be a field in the class, or if you can compute it from existing fields, a getter in the class